helios-client/src/main/java/com/spotify/helios/client/DefaultRequestDispatcher.java

```diff
@@ -183,7 +183,15 @@ private HttpURLConnection connect(final URI uri, final String method, final byte
         }
       }
     } catch (Exception e) {
-      log.warn("Couldn't get identities from ssh-agent", e);
+      log.debug("Couldn't get identities from ssh-agent", e);
+      log.warn(
+          "Your Helios cluster is setup to work over HTTPS based on the Helios DNS records "
```

Comment:
HTTPS and the client-side certificates are two different things though, right? Having HTTPS-configured without using client-certs seems like a valid setup.